src/main/java/com/joejoe2/chat/models/PrivateChannel.java

```diff
@@ -30,28 +27,30 @@ public class PrivateChannel extends TimeStampBase {
       name = "private_channels_users",
       joinColumns = {@JoinColumn(name = "private_channel_id", nullable = false)},
       inverseJoinColumns = {@JoinColumn(name = "user_id", nullable = false)})
-  Set<User> members;
+  private Set<User> members;
+
+  @ManyToMany
+  @BatchSize(size = 128)
+  @JoinTable(
+      name = "private_channels_blockedBy",
+      joinColumns = {@JoinColumn(name = "private_channel_id", nullable = false)},
+      inverseJoinColumns = {@JoinColumn(name = "user_id", nullable = false)})
+  private Set<User> blockedBy = new HashSet<>();
 
   @OneToMany(cascade = CascadeType.ALL, mappedBy = "channel", orphanRemoval = true)
-  List<PrivateMessage> messages;
+  private List<PrivateMessage> messages;
```

Comment:
where are getters & setter for these fields?

Reply:
Lombok will do that

Reply:
OK, let's rely on Lombok...